### ISSUE TITLE
Updating to 1.8.0 breaks gatsby 2.x compatibility

### ISSUE THREAD

**Issue by 50bbx** [mdreizin/gatsby-plugin-robots-txt#692]
Hello,

Because of the change in the package.json requiring a peerDependency of gatsby 5.0.0, this plugin breaks on a Gatsby 2 website.

As you can see here v1.7.1 is compatible with Gatsby 2, but 1.8.x is not.

https://github.com/mdreizin/gatsby-plugin-robots-txt/blob/v1.7.1/package.json

**Comment by grahamd711** [comment]
@mdreizin +1 for this issue. We are on Gatsby 3.14.6 and have the same issue installing packages due to the Gatsby 5.0.0 dependency.

**Comment by julian-miracle** [comment]
+1 too. We are on gatsby 4.24.7 and have the same issue installing packages due to the Gatsby 5.0.0 dependency. We are using v1.7.1 now

**Comment by OstaOleh** [comment]
We have the same issue. Gatsby 3.14.6 and Node 16v do not allow install packages. We can not downgrade not cuz it could break other packages.

**Comment by Primajin** [comment]
This should either be made backwards compatible or it should be clearly stated as a breaking change and thus a major version bump would be in order 😕

**Comment by davideghz** [comment]
it would be handy a table with a compatibility map: what plugin version is compatible with Gatsby 2.x, 3.x, 4.x, 5.x, ...

**Comment by rosswaycaster** [comment]
@davideghz I wish every Gatsby plugin had this. Upgrading a Gatsby 3.x site to even just 4.x has been a complete nightmare.

**Comment by timothy-kodes** [comment]
Yes you cannot update a package from 1.7.1 to 1.8.0, which according to semver should be a non-breaking change, and so suddenly require gatsby 5. This is my case because I am using Gatsby 4.25.7 and 1.8.0 depends on gatsby 5 suddenly.
